apps/vmware/connector/mode/alarmdatacenter.pm

```diff
@@ -223,6 +224,8 @@ sub manage_selection {
         $self->{datacenter}->{$datacenter_name} = { display => $datacenter_name, alarm => {}, dc_metrics => { 1 => { name => $datacenter_name } } };
         
         foreach (keys %{$response->{data}->{$datacenter_id}->{alarms}}) {
+            next if (defined($self->{option_results}->{filter_type}) && $self->{option_results}->{filter_type} ne '' &&
+                $response->{data}->{$datacenter_id}->{alarms}->{$_}->{type} =~ /$self->{option_results}->{filter_type}/);
```

Comment:
To get the same logic, it should be ```!=```

Reply:
okay I thought we wanted to exclude, but yes we can keep only the alarms of specified type(s)

Reply:
You can add an option to exclude also